monkey/agent_plugins/exploiters/mssql/config-schema.json

```diff
@@ -0,0 +1,60 @@
+{
+    "required": [
+        "target_ports",
+        "query_timeout",
+        "agent_binary_download_timeout",
+        "login_timeout",
+        "query_buffer_time"
```

Comment:
"Default" means, "this value will be used if no value is provided." Therefore, any property that has a default is, by definition, not required.